core/templates/pages/topic-editor-page/topic-editor-page.directive.html

```diff
@@ -1,5 +1,5 @@
 <div>
-  <div ng-if="$ctrl.getActiveTabName() === 'main'">
+  <div ng-if="$ctrl.getActiveTabName() === 'main' && $ctrl.getEntityType() === 'topic'">
```

Comment:
The main editor displays the thumbnail based on the page context. While navigating between tabs, the directive renders before window.location has changed and this causes the getEntityType to return an old value (based on the previous tab) causing an incorrect image to be fetched. To fix this, the directive should be rendered only after the correct entity type is returned.